core/sql/exp/ExpHbaseInterface.cpp

```diff
@@ -842,7 +842,9 @@ Lng32 ExpHbaseInterface_JNI::insertRow(
     transID = getTransactionIDFromContext();
   retCode_ = client_->insertRow((NAHeap *)heap_, tblName.val, hbs_,
                                 useTRex_, transID, rowID, row, timestamp, 
-                                checkAndPut, asyncOperation, useRegionXn, &htc);
+                                checkAndPut, asyncOperation, useRegionXn, 
+				0, // checkAndPut is false, so colIndexToCheck is not used
```

Comment:
How will the insertRow method distinguish between the "not used" case and a column index of zero? (Elsewhere I saw a comment that the column index is 0-based.)

Reply:
The java code in HTableClient::putRow has this line of code which will ensure that the number passed in for colIndexToCheck is meaningless when checkAndPut is FALSE. -1 would have been easier to read. I can make that change later, if you agree.

if (checkAndPut && colIndex == colIndexToCheck) {

Reply:
Thanks; code change looks good to me.